cms/apps/pages/templatetags/pages.py

```diff
@@ -7,22 +7,15 @@
 
 from cms.apps.pages.models import Page
 from cms.models import SearchMetaBase
+from django_jinja import library
+
+import jinja2
 
 register = template.Library()
 
 
 # Navigation.
-
-@register.inclusion_tag("pages/navigation.html", takes_context=True)
-def navigation(context, pages, section=None):
-    """
-    Renders a navigation list for the given pages.
-
-    The pages should all be a subclass of PageBase, and possess a get_absolute_url() method.
-
-    You can also specify an alias for the navigation, at which point it will be set in the
-    context rather than rendered.
-    """
+def navigation_entries(context, pages, section=None, is_json=False):
```

Comment:
This is a function that is just used in 2 templatetags and isn't actually used in templates. 

https://github.com/onespacemedia/cms/pull/111/files/9bd656b6eaac2db7de6214705b431eef8ed528b5#diff-9baa15ecda158c9c851d5a7b6747a500R61

For example uses it. We could prefix it with `get_` but that kind of conflicts with template based functions that use the `get_` prefix